package.json

```diff
@@ -24,33 +24,34 @@
     "typechain": "typechain --target=ethers-v6",
     "types:openzeppelin": "yarn typechain --out-dir src/contracts/openzeppelin ./node_modules/@openzeppelin/contracts/build/contracts/ERC20.json",
     "types:safe": "yarn typechain --out-dir src/contracts/safe ./node_modules/@safe-global/safe-deployments/dist/assets/**/*.json",
-    "postinstall": "husky install && yarn types:openzeppelin && yarn types:safe"
+    "postinstall": "yarn types:openzeppelin && yarn types:safe"
   },
   "dependencies": {
     "@gelatonetwork/relay-sdk": "^3.5.0",
+    "@nestjs/cli": "^9.4.2",
```

Comment:
Dependencies were updated with this move. This was unintended and I will revert the version bumps.